packages/slate-plain-serializer/package.json

```diff
@@ -17,24 +22,16 @@
     "slate": "^0.32.1"
   },
   "devDependencies": {
-    "babel-cli": "^6.10.1",
-    "browserify": "^13.0.1",
     "mocha": "^2.5.3",
     "slate": "^0.32.2",
-    "slate-hyperscript": "^0.5.2",
-    "uglify-js": "^2.7.0"
+    "slate-hyperscript": "^0.5.2"
   },
   "scripts": {
-    "build": "babel --out-dir ./lib ./src",
-    "build:max": "mkdir -p ./dist && NODE_ENV=production browserify ./src/index.js --transform babelify --transform envify --transform [ browserify-global-shim --global ] --standalone SlatePlainSerializer > ./dist/slate-plain-serializer.js",
-    "build:min": "mkdir -p ./dist && NODE_ENV=production browserify ./src/index.js --transform babelify --transform envify --transform [ browserify-global-shim --global ] --transform uglifyify --standalone SlatePlainSerializer | uglifyjs > ./dist/slate-plain-serializer.min.js",
-    "clean": "rm -rf ./dist ./lib ./node_modules",
-    "prepublish": "yarn run build && yarn run build:max && yarn run build:min",
-    "watch": "babel --watch --out-dir ./lib ./src --source-maps inline"
+    "clean": "rm -rf ./dist ./node_modules"
   },
-  "browserify-global-shim": {
+  "umdGlobals": {
     "immutable": "Immutable",
-    "slate": "Slate"
+    "slate": "slate"
```

Comment:
Should be capital `"Slate"` I think?

Reply:
Yep they should all be capitalized, I'll fix